main.js

```diff
@@ -160,4 +160,216 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+console.log("----- opdracht 1a ----");
+const brandsTv = inventory.map((inventory) => {
+return inventory.type
+});
+console.log(brandsTv);
+
+console.log("----- opdracht 1b ----");
+const soldOut = inventory.filter((inventory) => {
+  return inventory.originalStock - inventory.sold === 0;;
```

Comment:
Je kan dit nog iets korter schrijven
```suggestion
  return inventory.originalStock === inventory.sold;
```